utils/devel/rebuild_pot.sh

```diff
@@ -29,12 +29,12 @@ EOF
 
 # EXTRACT STRINGS AND CREATE POT
 find . -name '*.pl' -o -name '*.pm' | \
-  grep -v '\(b\|xt/\)lib/\|UI/pod/' | grep -v LaTeX | sort | \
+  grep -v '\(b\|xt/\)lib/\|UI/pod/\|\./\.[a-z]*\+' | grep -v LaTeX | sort | \
   utils/devel/extract-perl >> locale/LedgerSMB.pot
 
 find UI/ templates/ t/data/ \
      -name '*.html' -o -name '*.tex' -o -name '*.csv' | \
-   grep -v 'UI/\(js\|pod\)/)' | sort | \
+   grep -v 'UI/\(js\|pod\)' | sort | \
```

Comment:
I think the closing paren was indeed a mistake, but don't you think the forward slash was there for the reason that it wants to exclude files in those directories, but not the ones in `UI/js-src/`?

Reply:
Absolutely. Misread it for a backslash